### PR TITLE
fix: investigate missing resource issue

### PR DESCRIPTION
**Pull request by Maxim-Gadalov**
#256

### CODE REVIEW

**Comment by Maxim-Gadalov** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-core-pr-258.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Maxim-Gadalov** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-core-pr-258.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Maxim-Gadalov** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-core-pr-258.nightly-test.deltixhub.io
>E2E tests status: success

File: src/main/java/com/epam/aidial/core/service/ResourceService.java [hunk 1]
```diff
@@ -99,6 +99,7 @@ public ResourceService(Vertx vertx,
         this.cacheExpiration = Duration.ofMillis(cacheExpiration);
         this.compressionMinSize = compressionMinSize;
         this.prefix = prefix;
+        this.resourceQueue = BlobStorageUtil.toStoragePath(prefix, "resource:queue");
```
Comment (artsiomkorzun) [inline src/main/java/com/epam/aidial/core/service/ResourceService.java]: make similar to redisKey -> resource:prefix/queue

Comment (Maxim-Gadalov) [reply src/main/java/com/epam/aidial/core/service/ResourceService.java]: fixed

**Review by artsiomkorzun — APPROVED** [review]

**Pull request merged**